katello_certs_tools/fileutils.py

```diff
@@ -260,7 +258,8 @@ def makedirs(path,  mode=0755, user=None, group=None):
             # Changing permissions failed; ignore the error
             sys.stderr.write("Changing owner for %s failed\n" % dirname)
 
-def createPath(path, user='apache', group='apache', chmod=0755):
+
+def createPath(path, user='apache', group='apache', chmod='0755'):
```

Comment:
`os.mkdir` (as used in line 252) by `makedirs` does not accept string parameters (at least not in Python 3.6 and 3.7)